scrabble/__init__.py

```diff
@@ -0,0 +1,48 @@
+import check50
+import check50.c
+
+@check50.check()
+def exists():
+    """scrabble.c exists"""
+    check50.exists("scrabble.c")
+
+@check50.check(exists)
+def compiles():
+    """scrabble.c compiles"""
+    check50.c.compile("scrabble.c", lcs50=True)
+
+@check50.check(compiles)
+def tie_letter_case():
+    """handles letter cases correctly"""
+    check50.run("./scrabble").stdin("LETTERCASE").stdin("lettercase").stdout("Tie!").exit(0)
+
+@check50.check(compiles)
+def tie_punctuation():
+    """handles punctuation correctly"""
+    check50.run("./scrabble").stdin("Punctuation!?!?").stdin("punctuation").stdout("Tie!").exit(0)
+
+@check50.check(compiles)
+def test1():
+    """handles specification test 1"""
```

Comment:
For these, instead of "test 1"-"test 4", let's use a more descriptive "correctly identifies WORD1 as winner over WORD2" or some wording to that effect, to facilitate students testing on their own.